src/Backend/Modules/MediaLibrary/Actions/MediaItemEdit.php

```diff
@@ -75,12 +77,12 @@ public function execute(): void
             $this->getBackLink(
                 [
                     'report' => 'media-item-edited',
-                    'var' => $updateMediaItem->title,
+                    'var' => $updateMediaItem->url,
```

Comment:
why did you change this?

Reply:
Before
MediaItem->getTitle();

After
MediaItem->getTranslation(locale)->getTitle();

That is why I'm now using $updateMediaItem->url, because ->title doesn't exist anymore